src/database/README.md

```diff
@@ -0,0 +1,96 @@
+# Mongo DataBase :leaves:
+
+## 1. Why using MongoDB ? :thinking: 
+
+Reasons why we choosed MongoDB: :woman_teacher: 
+* Documents database: the data structure is close to the objects we need in the code.
+* Polymorphic Schema: Mongo makes it easy to update schema without doing any migration scripts.
+* If the project grows well, mongoDB have great reading performances (we do not need writing to be lighting fast).
+* Great community.
+
+## 2. Schema definition
+
+### 2.1 **Authors** collection :writing_hand: 
+
+
+```js
+{
+  _id: MONGOID,
+  firstName: String,
+  lastName: String,
+  birth: Date,
+  death: Date,
+  nationality: String,
+  biography: Text,
+  movements: [movementsMongoIds],
+  works: [worksMongoIds]
+}
+```
+
+### 2.2 **Works** collection :book:
+
+```js
+{
+  _id: MONGOID,
+  authorId: MONGOID,
+  title: String,
+  parutionDate: Date,
+  language: String,
+  publisher: String,
+  rawFormat: String, // epub, mobi, csv 
+  type: String,      // poem, historical story, novel..
+  extracted: {},     // ML stuff extracted
+  content: Text
```

Comment:
Maybe we can store raw text content on S3 or similar storage and have an URL to make thelink here in mongo